modules/tensor_mechanics/include/materials/ComputeElasticityBeam.h

```diff
@@ -0,0 +1,43 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#ifndef COMPUTEELASTICITYBEAM_H
+#define COMPUTEELASTICITYBEAM_H
+
+#include "Material.h"
+
+/**
+ * ComputeElasticityBeam defines the elasticity vector for the beam element
```

Comment:
Since 'elasticity vector' isn't a standard thing, maybe better to explain that a little more.  I'd say something like:

ComputeElasticityBeam computes the equivalent of the elasticity tensor for the beam element, which are vectors of material translational and flexural stiffness.